cyences_app_for_splunk/appserver/static/dashboard.js

```diff
@@ -80,4 +80,22 @@ require([
     // Handles the multi-select option properly
     SplunkCommonUtilities.vSetupMultiSelectHandlerOnAll();
     // When user selects any item, it will automatically unselect "All" option. And when user selects "All" option then it will automatically unselect all other.
+
+    ProductName = SplunkCommonUtilities.VTokenManagerObj.getToken('tkn_product_name_for_data_availability_search');
+
+    var searchQuery = '| cyencesproductmanager operation="buildproductspecificsearches" product_name="' + ProductName + '" | table data_availablity_panel_search';
```

Comment:
Bro, shall we handle the case where dashboard does not have the token for product??

Reply:
It was missed, Let me handle that.